.github/workflows/ci.yml

```diff
@@ -375,3 +375,57 @@ jobs:
           else
             echo "Integration tests passed"
           fi
+  smoke-test-released-exemplar:
```

Comment:
I think this should be removed from ci.yaml, as it is duplicated in smoketest.yaml, which also runs on PRs?